fluent-bit-cloudwatch.go

```diff
@@ -85,6 +85,8 @@ func getConfiguration(ctx unsafe.Pointer, pluginID int) cloudwatch.OutputPluginC
 	logrus.Infof("[cloudwatch %d] plugin parameter endpoint = '%s'\n", pluginID, config.CWEndpoint)
 	config.CredsEndpoint = output.FLBPluginConfigKey(ctx, "credentials_endpoint")
 	logrus.Infof("[cloudwatch %d] plugin parameter credentials_endpoint = %s\n", pluginID, config.CredsEndpoint)
+	config.LogFormat = output.FLBPluginConfigKey(ctx, "log_format")
```

Comment:
Are there going to be other log formats?

Reply:
In the future we may support other log format values either for metrics or field extraction.